src/app/pages/home/footer/footer.component.html

```diff
@@ -1 +1,31 @@
-<div>Footer Page</div>
\ No newline at end of file
+
+<nav class=" navbar navbar-dark bg-dark mt-10 clearfix">
+
+    <div class="navbar-expand m-auto navbar-text">

```

Comment:
Don't forget to remove `navbar-expand` and `m-auto` from here once you've done the changes above.
```suggestion
    <div class="navbar-text">
```